src/guiengine/skin.cpp

```diff
@@ -1271,9 +1271,10 @@ void Skin::drawSpinnerBody(const core::recti &rect, Widget* widget,
             params=&SkinConfig::m_render_params["spinner4::neutral"];
         else
         {
-            Log::fatal("Skin::drawSpinnerBody", "Unknown playerID (more than 4 players?)");
+            //TODO Add individual spinner skins
```

Comment:
This TODO will have to be resolved before this is merged

Reply:
I don't think it should come in the way of a merge, but definitely should be left as an issue, or something on an agenda to do for the next release milestone